docs/configuration.md

```diff
@@ -16,41 +16,46 @@ Unless you need to set a non-default value, it is recommended to only populate o
+| `node.log_level`                               | info                                                 | The log level of the validator. Possible values: `info`, `debug`, `trace` case insensitive.                                                                                                                                                                                                                                                                                                                                                 |
+| `node.port`                                    | 5200                                                 | The port on which the application runs.                                                                                                                                                                                                                                                                                                                                                                                                     |
+| `node.validator`                               | true                                                 | The primary mode in which the application will run. If set to `true`, the application will make write operations (HCS submission, Scheduled Transactions). If set to `false`, the application will be in a read-only mode, searching for transactions/messages from the other validators in the networks.                                                                                                                                   |
 
 Configuration for `config/bridge.yml`:
 
 | Name                                                          | Default | Description                                                                                                                                                                                                                                                          |
 |---------------------------------------------------------------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `bridge.use_local_config`                                     | false   | If `use_local_config` is `false` it keep using the local config, if it is `true` it will fetch the bridge config from the topics and will start watcher to keep it updated.                                                                                          |
```

Comment:
If `use_local_config` is `true` it keeps using the local config, if it is `false` it will fetch the bridge config from the topics and will start watcher to keep it updated.